src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Completion/CompletionListMerger.cs

```diff
@@ -121,8 +130,8 @@ private static void TrySplitJsonElement(object data, ref PooledArrayBuilder<Json
             return;
         }
 
-        if ((jsonElement.TryGetProperty(s_data1Key, out _) || jsonElement.TryGetProperty(s_data1Key.ToLowerInvariant(), out _)) &&
-            (jsonElement.TryGetProperty(s_data2Key, out _) || jsonElement.TryGetProperty(s_data2Key.ToLowerInvariant(), out _)))
+        if ((jsonElement.TryGetProperty(Data1Key, out _) || jsonElement.TryGetProperty(Data1Key.ToLowerInvariant(), out _)) &&
```

Comment:
I know you didn't change this behavior, but why can't we be deterministic about casing for these property names?

Reply:
Yeah you know what, this does seem overly paranoid, especially now that we specify the attributes and we know the whole stack uses STJ